### DISCUSSION TITLE
Github account got flagged as spam after mentioning users in a Github Discussion

### DISCUSSION THREAD

**Discussion by ioan-faur** [community/community#45341] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Can someone kindly help with my github account `fioan89` which got flagged a few weeks ago as spam? I had to create a separate account for this discussion to be visible :(

For a bit of context I was trying to have a discussion around a personal project and gather feedback from all the users that contributed with issues, feature requests and PR's to the project. My mistake was to mention all these users in the discussion. A couple of them reported the discussion to Github as spam. I've read Github's Code of Conduct before and after posting the discussion. I'm not sure what rule I have broken. I have not used any foul language, I apologized a couple of times for mentioning users that might no longer have an interest in the project, I tried to be respectful and I haven't revealed any private information like emails, names, etc.. I just mentioned Github usernames in a public discussions, usernames which in the past contacted me in one way or another. The discussion that started all this can be found here: https://github.com/fioan89/sourcesync/discussions/103

I'd like to think that open source is not a one way street, but after this incident I'm not sure what to think any longer :(

I requested an account review two weeks ago (request #1955747) but no feedback since. My actions on github are very limited, which hinders the development of the plugin, which directly impacts the whole community around the project.

I'm grateful for any help/support on this matter. Thank you and please don’t hesitate to let me know if you have any questions or concerns.

Cheers,
Ioan Faur

**Comment by queenofcorgis** [comment]
Hi there! 

The community is unable to assist with account-specific matters like this one so I'll be locking this post. However, I did check in on your ticket and it looks like you received a response on January 10th. For any follow-ups please continue to submit tickets. 

Cheers!
